app/code/СhaOS/AskQuestion/view/frontend/requirejs-config.js

```diff
@@ -1,8 +1,17 @@
 var config = {
+    config: {
```

Comment:
This task is not completed:
1) a part of the files was in the previous home task for some reason
2) the script `askQuestion.js` DOES NOT contain any logic related to collecting form data, submitting it, working with cookies and so on. 

The task is half-ready. Please, complete it before creating pull requests.